S15-nfg/GraphemeBreakTest.t

```diff
@@ -64,7 +66,7 @@ sub MAIN (Str:D :$file = $location, Str :$only, Bool:D :$debug = False) {
     die "Can't find file at ", $file.IO.absolute unless $file.IO.f;
     note "Reading file ", $file.IO.absolute;
     my @fail;
-    plan (1943);
+    plan (1706); #1943
```

Comment:
Did the number of tests in the file decrease by over 200 between the versions?

Reply:
I ran testing and changed the plan to reflect the number of tests it reported doing. I didn't edit any of the generated test files by hand. (By the same token, I didn't open each file and count how many tests were in each, I took what prove reported.)

Reply:
Checked myself. Looks good.